### ISSUE TITLE
Access to original Masonry Events

### ISSUE THREAD

**Issue by austriker27** [shershen08/vue-masonry#90]
This package doesnt seem to access the [masonry events](https://masonry.desandro.com/events.html), it would be great if that could be added. 

If they are included, some documentation on how to use would be great. 

Thanks

**Comment by shershen08** [comment]
You are right events from original library are not available at the moment
this could be a nice addition to v.2.0 indeed

**Comment by dawin09** [comment]
Hello @shershen08 is this still on your roadmap, would be nice to have the events.

**Comment by kjh4357** [comment]
I want the layout complete function to be added.
